server/server_test.go

```diff
@@ -397,9 +427,13 @@ func TestLogSourceIPs(t *testing.T) {
 	go server.Run()
 	defer server.Shutdown()
 
+	require.Empty(t, fake.sourceIPs)
+
 	req, err := http.NewRequest("GET", "http://127.0.0.1:9195/error500", nil)
 	require.NoError(t, err)
 	http.DefaultClient.Do(req)
+
+	require.NotEmpty(t, fake.sourceIPs)
```

Comment:
Instead of testing for `NotEmpty`, the confidence level would be higher if would test for equality on the expected value.

Reply:
The test was changed to check for the IP address.